### PR TITLE
1Updated Doc

### PR DESCRIPTION
**Pull request by AasthaDakhera**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # CraftAPIClient
-
+# A change
```
Comment (AasthaDakhera) [inline README.md:2]: add change

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # CraftAPIClient
```
Comment (Diiksha-agarwal) [inline README.md:1]: maja nahi aaya

Comment (AasthaDakhera) [reply README.md:1]: then you do the changes

Comment (Diiksha-agarwal) [reply README.md:1]: ye tune kiya bhi nahi tha

**Pull request merged**